jupyterlab/packages/jupyterlab-jupytext-extension/src/factory.ts

```diff
@@ -68,6 +68,12 @@ export function createFactory(
   });
   docRegistry.addWidgetFactory(factory);
 
+  // Add the widget extensions of the 'Notebook' factory to the 'Jupytext Notebook'
+  // factory.
+  for (const extension of docRegistry.widgetExtensions('Notebook')) {
+    docRegistry.addWidgetExtension(FACTORY, extension);
+  }
```

Comment:
Is there a race condition here? I mean, what happens if there are widgets extensions that have registered themselves to `Notebook` factory **after** Jupytext?

Reply:
Actually, I missed that there was a signal when a widget extension is added to the registry.
We should probably use it in the factory to keep the factory up to date.

Reply:
Yess, that would be way to go if we can make Jupytext subscribe to that signal.

Reply:
I updated the PR